### PR TITLE
Added Turkish translation and some other i18n work.

### PR DESCRIPTION
**Pull request by ismailcarlik**

### CODE REVIEW

File: Plugins/Wox.Plugin.Sys/Main.cs [hunk 1]
```diff
@@ -207,17 +210,6 @@ private List<Result> Commands()
                         context.API.OpenSettingDialog();
                         return true;
                     }
-                },
-                new Result
-                {
-                    Title = "Reload Plugin Data",
```
Comment (jjw24) [inline Plugins/Wox.Plugin.Sys/Main.cs:213]: Please reverse this, probably accidentally removed during your merge. This is from PR https://github.com/jjw24/Wox/pull/50

**Comment by jjw24** [comment]
Looking really good! Please update per the one comment above, looks like an accidental removal during your merge.

File: Plugins/Wox.Plugin.Sys/Main.cs (hunk 1 above)
Comment (ismailcarlik) [reply Plugins/Wox.Plugin.Sys/Main.cs:213]: done ✔
also made the subtitle translateable.

Comment (jjw24) [reply Plugins/Wox.Plugin.Sys/Main.cs:213]: awesome thanks!

**Review by jjw24 — APPROVED** [review]

**Pull request merged**